.github/workflows/microspeech.yaml

```diff
@@ -0,0 +1,217 @@
+          while read artifact; do
+            IFS=$'\t' read name url <<< "$artifact"
+            gh api $url > "$name.zip"
+            unzip -d "$name" "$name.zip"
+          done
+
+          cp ARM.CMSIS-NN.*.pack ../Platform_FVP_Corstone_SSE-300_Ethos-U55/
+
+      - name: Download latest release pack
+        if: ${{ ! github.event.workflow_run }}
+        env:
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+        run: |
+          gh -R ${{ github.repository }} release download -p "*.pack" -D Platform_FVP_Corstone_SSE-300_Ethos-U55
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1-node16
+        with:
+          role-to-assume: ${{ env.AWS_ASSUME_ROLE }}
+          aws-region: ${{ env.AWS_DEFAULT_REGION }}
+
+      - name: Apply patch
+        run: |
+          VERSION=$(\
+            basename $(ls Platform_FVP_Corstone_SSE-300_Ethos-U55/ARM.CMSIS-NN.*.pack) | \
+            cut -d. -f 3-5 | cut -d+ -f 1 \
+          )
+          patch -p0 <<EOF
+          --- Platform_FVP_Corstone_SSE-300_Ethos-U55/microspeech.Example.Reference.cprj
+          +++ Platform_FVP_Corstone_SSE-300_Ethos-U55/microspeech.Example.Reference.cprj
+          @@ -11,2 +11,3 @@
+            <packages>
+          +    <package name="CMSIS-NN" vendor="ARM" version="${VERSION}:${VERSION}"/>
+               <package name="CMSIS" vendor="ARM" version="5.9.0:5.9.0"/>
+          @@ -130,3 +131,2 @@
+                 <file category="sourceCpp" name="../micro_speech/src/recognize_commands.cc"/>
+          -      <file category="sourceC" name="../micro_speech/src/arm_nn_softmax_common_s8.c"/>
+               </group>
+          --- Platform_FVP_Corstone_SSE-300_Ethos-U55/microspeech.Example.SIMD.cprj
+          +++ Platform_FVP_Corstone_SSE-300_Ethos-U55/microspeech.Example.SIMD.cprj
+          @@ -11,2 +11,3 @@
+            <packages>
+          +    <package name="CMSIS-NN" vendor="ARM" version="${VERSION}:${VERSION}"/>
+               <package name="CMSIS" vendor="ARM" version="5.9.0:5.9.0"/>
+          @@ -130,3 +131,2 @@
+                 <file category="sourceCpp" name="../micro_speech/src/recognize_commands.cc"/>
+          -      <file category="sourceC" name="../micro_speech/src/arm_nn_softmax_common_s8.c"/>
+               </group>
+          --- Platform_FVP_Corstone_SSE-300_Ethos-U55/microspeech.Example.Helium.cprj
+          +++ Platform_FVP_Corstone_SSE-300_Ethos-U55/microspeech.Example.Helium.cprj
+          @@ -11,2 +11,3 @@
+            <packages>
+          +    <package name="CMSIS-NN" vendor="ARM" version="${VERSION}:${VERSION}"/>
+               <package name="CMSIS" vendor="ARM" version="5.9.0:5.9.0"/>
+          @@ -130,3 +131,2 @@
+                 <file category="sourceCpp" name="../micro_speech/src/recognize_commands.cc"/>
+          -      <file category="sourceC" name="../micro_speech/src/arm_nn_softmax_common_s8.c"/>
```

Comment:
Wondering why this is needed if we are using the pack from the new repository. In CMSIS 5.9.0 release, the arm_nn_softmax_common_s8 file was missed in the pdsc. 
How does this fit in when using the new CMSIS-NN repo and its pack?

Reply:
This patch is exactly addressing this difference. The project is still using NN from CMSIS 5.9.0 pack. The patch updates the projects to use CMSIS-NN 4.0.0 pack instead.